iop/system/udnl/src/udnl.c

```diff
@@ -367,6 +383,36 @@ static volatile unsigned int *func_00000f80(volatile unsigned int *address)
 
 // 0x00001b38
 void func_00001b38(unsigned int arg1);
+__asm__
```

Comment:
Shouldn't be better if we use the `asm` code directly inside of the function? instead of defining the header and then implementing the "whole" function using ASM?

Reply:
That would be better, but I plan to do that in another pass (not in this PR) because it requires figuring out what registers are read/written/clobbered and that has a higher change of breaking.